packaging/windows/product.wxs.in

```diff
@@ -27,7 +27,6 @@
         <Condition Message="CodeReady Containers requires at least 9GB of RAM to run. Aborting installation.">
             <![CDATA[Installed OR (PhysicalMemory >= 9126)]]>
         </Condition>
-        <util:Group Id="HypervAdminsGroup" Name="Hyper-V Administrators" />

```

Comment:
The commit log could use a "why?" (to handle localized windows version where `Hyper-V Administrators`` is localized. wix does not provide a way to use the sid or to get the localised name of the group. This commit changes the installer to use crc setup to do this. This fixes a regression when transitioning from crc setup to the installer).

Reply:
Added this to the commit log